packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -567,22 +567,19 @@ function isDataObject(value: unknown): value is DataObject {
  * Tries to use `instanceof` because we decided that a version mix-up with
  * {@link @fluidframework/aqueduct#} is unlikely between devtools and end-user applications, and we don't support it anyway.
  * In addition, we check for the presence of key properties that make a `TreeDataObject` unique:
- * - {@link TreeDataObject#sharedTree | sharedTree} getter
- * - {@link TreeDataObject#treeView | treeView} getter
- * - {@link TreeDataObject#initializeInternal | initializeInternal} method
+ * - `TreeDataObject.tree` getter
```

Comment:
Can't we still use the `{@link}` syntax? Any reason we changed to using backticks here?